components/EditSidebar/index.js

```diff
@@ -0,0 +1,127 @@
+import Portal from 'react-portal'
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { css } from 'glamor'
+import { colors } from '@project-r/styleguide'
+
+const panelWidthPx = 180
+const transitionDurationMs = 300
+
+const styles = {
+  container: css({
+    position: 'absolute',
+    top: 0,
+    right: '10px'
+  }),
+  toolbar: {
+    backgroundColor: '#fff',
+    borderLeft: `1px solid ${colors.divider}`,
+    borderRadius: '2px 0 0 2px',
+    bottom: 0,
+    width: `${panelWidthPx}px`,
+    opacity: 1,
+    padding: '40px 10px 10px',
+    position: 'absolute',
+    right: 0,
+    top: 0,
+    transition: `right ${transitionDurationMs}ms`,
+    zIndex: 1
+  },
+  closeButton: css({
+    padding: '10px',
+    position: 'absolute',
+    right: 0,
+    top: 0,
+    ':hover': {
+      opacity: 0.6
+    }
+  }),
+  openButton: css({
+    position: 'absolute',
+    right: 0,
+    top: '10px'
+  })
+}
+
+const CloseIcon = ({ size, fill, style }) =>
```

Comment:
that could probably save you some time, especially considering, that we've already added it. 
Just get the icon you want from:
http://gorangajic.github.io/react-icons/

and 

```
import CloseIcon from 'react-icons/lib/md/close'
```
Instead of `fill`, use `color`